howso/utilities/installation_verification.py

```diff
@@ -1,7 +1,13 @@
 from dataclasses import dataclass
-from datetime import datetime, timedelta
+from datetime import (
+    datetime,
+    timedelta
+)
```

Comment:
If the imports can fit on a single line under the max line length, lets do so

Reply:
Did a sweep of all the files I touched in the scope of this PR including ones I didn't screw around with import-wise. Linking diff is empty compared to main.

Addressed by: 7ed43ce

Reply:
Missed one and corrected in: 6139dc2